### PR TITLE
Reflector: Remove recursion parameter.

### LINKED ISSUE #15362: Reflector recursion is capped to 3 max
##### Description of the problem
When you set recursion level for reflectors, it is capped to three ( see the screenshot )

image

All side walls were replaced by reflectors following the [mirror example](https://github.com/mrdoob/three.js/blob/dev/examples/webgl_mirror.html) using 

```
			var geometry1 = new THREE.PlaneBufferGeometry( 100, 100 );
			var verticalMirror1 = new THREE.Reflector( geometry1, {
					clipBias: 0.003,
					textureWidth: WIDTH * window.devicePixelRatio,
					textureHeight: HEIGHT * window.devicePixelRatio,
					color: 0x889999,
					recursion: 1000
				} );
				verticalMirror1.position.y = 50;
				verticalMirror1.position.z = 50;
        verticalMirror1.rotateY( Math.PI );
				scene.add( verticalMirror1 );

			var geometry2 = new THREE.PlaneBufferGeometry( 100, 100 );
			var verticalMirror2 = new THREE.Reflector( geometry2, {
					clipBias: 0.003,
					textureWidth: WIDTH * window.devicePixelRatio,
					textureHeight: HEIGHT * window.devicePixelRatio,
					color: 0x889999,
					recursion: 1000
				} );
        verticalMirror2.position.x = - 50;
				verticalMirror2.position.y = 50;
				verticalMirror2.rotateY( Math.PI / 2 );
				scene.add( verticalMirror2 );
        
			var geometry3 = new THREE.PlaneBufferGeometry( 100, 100 );
			var verticalMirror3 = new THREE.Reflector( geometry3, {
					clipBias: 0.003,
					textureWidth: WIDTH * window.devicePixelRatio,
					textureHeight: HEIGHT * window.devicePixelRatio,
					color: 0x889999,
					recursion: 1000
				} );
        verticalMirror3.position.x = 50;
        verticalMirror3.position.y = 50;
        verticalMirror3.rotateY( - Math.PI / 2 );
				scene.add( verticalMirror3 );
```

##### Three.js version

Latest version from the repository

##### Browser

- [x] Chrome 64.0.3282.140 (Official Build) (64-bit) (cohort: 68_84_win)
- [X] Firefox Quantom 63.0.3 64-bit
- [X] Internet Explorer 11.0.95 (KB4466536)

##### OS

- [ ] All of them
- [X] Windows
- [ ] macOS
- [ ] Linux
- [ ] Android
- [ ] iOS

##### Hardware Requirements (graphics card, VR Device, ...)

Intel ID Graphics ( PCI\VEN_8086&DEV_0A16&SUBSYS_05E01028&REV_0B ) (PCI\VEN_8086&DEV_0A16&CC_030000 )

### PR DESCRIPTION
**Pull request by Mugen87**
see https://github.com/mrdoob/three.js/issues/15362#issuecomment-605984488

tl;dr: `recursion` has not effect since `Reflector` can't re-render itself.

Closes #15362.

### CODE REVIEW

**Comment by Mugen87** [comment]
https://raw.githack.com/Mugen87/three.js/dev48/examples/webgl_mirror.html

**Comment by munrocket** [comment]
Seems that webgl_materials_envmaps_exr screenshot still fails.

**Comment by mrdoob** [comment]
> tl;dr: `recursion` has not effect since `Reflector` can't re-render itself.

👍

**Comment by mrdoob** [comment]
Thanks!
